solutions/src/main/java/ru/sberbank/school/task02/Client.java

```diff
@@ -0,0 +1,134 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.ConverterConfigurationException;
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.exception.WrongSymbolException;
+import ru.sberbank.school.task02.util.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static ru.sberbank.school.task02.FxRequestConverter.*;
+
+/**
+ * Created by Mart
+ * 10.05.2019
+ **/
+public class Client implements FxClientController{
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-745520778>*